here-naksha-lib-psql/src/commonTest/kotlin/naksha/psql/TestPsql.kt

```diff
@@ -168,4 +168,85 @@ class TestPsql {
                     }
             }
     }
+
+    @Test
+    fun insert_and_update(){
+        // Given: Create collection request
+        val col = NakshaCollection("test_${randomString().lowercase()}")
+        val writeCollectionRequest = WriteRequest()
+        writeCollectionRequest.writes += Write().createCollection(null, col)
+
+        // And: create feature in collection request
+        val originalFeature = NakshaFeature().apply { id = "feature_1"
+            properties = NakshaProperties().apply {
+                featureType = "some_feature_type"
+            }
+        }
+        val writeFeaturesReq = WriteRequest().add(
+            Write().createFeature(null, col.id, originalFeature)
+        )
+
+        // And:
+        val featureToUpdate = NakshaFeature().apply {
```

Comment:
maybe changing the values of the original feature object and reuse here would be more resilient, in case in the future some randomly preset properties will be different for every NakshaFeature() and NakshaProperties()

Reply:
Changed to referential id